plugins/email/server/auth/email.ts

```diff
@@ -117,26 +114,6 @@ router.get(
       return ctx.redirect("/?notice=user-suspended");
     }
 
-    if (user.isInvited) {
```

Comment:
Moved this to a processor, as here we'll only send the `InviteAcceptedEmail` if our invited user joins with email. But, it's actually possible and likely they will sign-in with SSO – in which case this email is never sent.